src/components/BlockingViews/BlockingView.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import variables from '../../styles/variables';
+import Icon from '../Icon';
+import Text from '../Text';
+
+const propTypes = {
+    /** Expensicon for the page */
+    icon: PropTypes.func.isRequired,
+
+    /** Color for the icon (should be from theme) */
+    iconColor: PropTypes.string.isRequired,
```

Comment:
NAB – seems like we could default this to `themeColors.offline` ?

Reply:
Yeah I think that's fair. I'll add that in